src/index.js

```diff
@@ -4,12 +4,11 @@
 // TODO Do we want to support OAuth 1.0? https://github.com/indieocean/Titanium-OAuth-Client/blob/master/titanium_oauth.js
 // TODO Allow users to specify scopes when authorizing
 const CALLBACK_URL = 'http://localhost/Callback'; //'urn:ietf:wg:oauth:2.0:oob' // FIXME can we redirect to some other URL scheme that won't show errors in console?
-// TODO ALlow users to modify the look and feel of the window that opens the webview for auth? really doesn't matter much...
+// TODO Allow users to modify the look and feel of the window that opens the webview for auth? really doesn't matter much...
 // Only important one here is the title.
 const AUTH_WINDOW_OPTIONS = {
 	backgroundColor : 'white',
 	barColor : '#000000',
-	modal : true,
```

Comment:
I think we need this to still be modal by default so they don't switch out of it mid-login?